docs/development/contributing.md

```diff
@@ -105,9 +106,9 @@ be assigned by the Technical Board or Maintainers:
 | technical board | This label is given for PRs which will be discussed in a meeting of the Technical Board. The label will be removed after the discussion.                                  |
 
 <a name="rules-for-maintainers-assigned-to-prs"></a>
```

Comment:
Link name is wrong after merge:

<a name="rules-for-maintainers-assigned-to-prs"></a>